### PR TITLE
allowed float.div 0.0 (#1791)

### PR DESCRIPTION
**Pull request by maxonfjvipon**
Closes: #1791

### CODE REVIEW

**Comment by maxonfjvipon** [comment]
@Graur please take a look

File: eo-runtime/src/main/eo/org/eolang/nan.eo [hunk 1]
```diff
@@ -89,7 +97,7 @@
 
   # Converts this to bytes
   [] > as-bytes
-    7F-C0-00-00 > @
+    ^.@.as-bytes > @
```
Comment (Graur) [inline eo-runtime/src/main/eo/org/eolang/nan.eo:100]: @maxonfjvipon I don't get this part. Can you provide some short explanation?

**Review by Graur — CHANGES_REQUESTED** [review]

File: eo-runtime/src/main/eo/org/eolang/nan.eo (hunk 1 above)
Comment (maxonfjvipon) [reply eo-runtime/src/main/eo/org/eolang/nan.eo:100]: `7F-C0-00-00` - is float NaN (4 bytes). 
On the last meeting we decided that float in `eo` is a double in Java - 8 bytes. 
Double NaN of 8 bytes - `7F-F8-00-00-00-00-00-00` (this value also will be got as result of `0.0.div 0.0`)

I tried to hardcode this bytes values and tests passed on my machine, but they fails in CI for some reason (you can check one of the previous commits)

So (0.0.div 0.0).as-bytes is kinda safe

Comment (Graur) [reply eo-runtime/src/main/eo/org/eolang/nan.eo:100]: @maxonfjvipon I agree, thanks!

**Review by Graur — APPROVED** [review]
@maxonfjvipon Looks good to me, thanks!
@yegor256 Please, have a look

**Comment by maxonfjvipon** [comment]
@yegor256 please take a look

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**